common/smoke-test/BlobStorage.ts

```diff
@@ -25,16 +25,19 @@ export class BlobStorage {
 
     const account =
       process.env["STORAGE_ACCOUNT_NAME"] || "<YourStorageAccountNAME>";
+    const endpointSuffix =
```

Comment:
Should we consider combing the account and endpoint into the same variable?

Reply:
JS' examples all suggest using a `BlobServiceClient` -- https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-storage-blob/12.1.1/index.html#create-the-blob-service-client 

However, a `BlobClient` does exist which, from the docs, looks like it takes a connection string: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-storage-blob/12.1.1/classes/blobclient.html

Reply:
Further discussion with @jeremymeng revealed that there is a `fromConnectionString` method on `BlobServiceClient` and using that puts the environment variable requirements much more in line with the other smoke tests. -- https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-storage-blob/12.1.1/classes/blobserviceclient.html#fromconnectionstring

The JS library is done this way because there wasn't an easy way to differentiate the multiple constructors.